bluemira/codes/plasmod/equilibrium_2d_coupling.py

```diff
@@ -666,7 +667,10 @@ def calc_curr_dens_profiles(
         dum3 = np.gradient(dum2, psi_1D)
         betahat = dum3 / q3 / AA
         chat = -4 * np.pi**2 * MU_0 * pprime / AA
-        FF = np.sqrt(2.0 * y)  # noqa: N806
+        #  or np.where(y >= 0, np.sqrt(2.0 * y), -np.sqrt(2.0 * abs(y)))
+        #  or np.sqrt(2.0 * abs(y))
+        FF = np.sqrt(2.0 * np.clip(y, EPS, None))  # noqa: N806
```

Comment:
Actually, why clip at `EPS` and not at `0.0`?

Reply:
no reason really just habit not using 0. I'll change, see what happens

Reply:
still didnt converge on the equilibria example put probably more correct so pushed